assets/js/commento.js

```diff
@@ -204,7 +207,7 @@
 
     var _showdownUrl = "/assets/showdown.min.js";
     var _spectreUrl = "/assets/spectre.min.css";
-    var _commentoCssUrl = "/assets/commento.min.css";
+    var _commentoCssUrl = "/assets/style/commento.min.css";
```

Comment:
just seen the js folder is `/assets/js/`, in that case I think `./css` would trump `./style` :man_shrugging: